core/Container/ContainerFactory.php

```diff
@@ -82,4 +82,16 @@ private function addEnvironmentConfig(ContainerBuilder $builder)
 
         $builder->addDefinitions($file);
     }
+
+    private function addPluginConfigs(ContainerBuilder $builder)
+    {
+        $plugins = Config::getInstance()->Plugins['Plugins'];
```

Comment:
Couldn't use the plugin manager here because `getActivatedPlugins()` returned an empty array (plugins weren't loaded yet).